### ISSUE TITLE
Trial of V2

### ISSUE THREAD

**Issue by GerBreOwn** [darklow/django-suit#697]
I saw that V2 is available for testing.  

So, I installed it but did NOT like it as it was giving me errors so I uninstalled it and reinstalled 0.2.6 but it did NOT go back to what I had before. 

Now the time, welcome, etc which were at the right side of screen are now scrunched at the left side.  The menu options that were at the left side are now at the left bottom.  It now looks UGLY!

Is it possible that V2 did NOT get completely removed that is causing the problems?  If so how can I completely remove it and reinstall V0.2.26?  BTW, what happened to V1, going from V0 to V2?

Thanks

**Comment by darklow** [comment]
Not sure, try cleaning up `pyc` files:
```
find . -name "*.pyc" -exec rm -f {} \;
```
Make sure you run collecstatic if not serving files by runserver, make sure you run correct version with pip freeze

**Comment by GerBreOwn** [comment]
After posting the above issue I started getting another which I posted as "


  TemplateSyntaxError at /admin/login/ #698"

Now I am getting errors "


  TemplateDoesNotExist at /admin/login/

admin/login.html.

This is showing up at 3 places per the following error messages: "

Using engine |django|:

  * |django.template.loaders.app_directories.Loader|:
    /media/Projects/simonproj/visit/templates/admin/login.html (Source
    does not exist)
  * |django.template.loaders.app_directories.Loader|:
    /home/gerald/.virtualenvs/simonproj/lib/python3.6/site-packages/admin_reports/templates/admin/login.html
    (Source does not exist)
  * |django.template.loaders.app_directories.Loader|:
    /home/gerald/.virtualenvs/simonproj/lib/python3.6/site-packages/django/contrib/auth/templates/admin/login.html
    (Source does not exist)

Not sure if they have anything to do with Suit or not.

I removed all of the *pyc files but when I ran runserver some of them 
were recreated.

In my requirements.txt file I have django-suit without any version and 
when that is run version 0.2.6 gets installed.  When I tried the 
development version it installed 0.2.5.

A further question: I am developing my application for a doctor to be 
used in his clinic to maintain his patients records.  Will this still 
qualify for the FREE license?  The program will NOT be used to generate 
any income.

Thanks for any further information on how to solve this problem.

Gerald Brown
